iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.hpp

```diff
@@ -34,6 +34,8 @@ namespace popo
 enum class AllocationError
 {
     INVALID_STATE,
+    NO_MEMPOOLS_AVAILABLE,
+    NO_MEMPOOL_FOR_REQUESTED_CHUNK_SIZE,
```

Comment:
I'm not sure if we need both or could just get along with `NO_MEMPOOLS_AVAILABLE`?